langchain4j-pinecone/src/main/java/dev/langchain4j/store/embedding/pinecone/PineconeEmbeddingStore.java

```diff
@@ -114,67 +120,53 @@ public List<String> addAll(List<Embedding> embeddings, List<TextSegment> textSeg
         return ids;
     }
 
+    @Override
+    public void removeAll(Collection<String> ids) {
```

Comment:
please create `PineconeEmbeddingStoreRemovalIT` that extends from `EmbeddingStoreWithRemovalIT` (you can override tests for unsupported removal methods)

Reply:
Done.